qa-tests-backend/src/main/groovy/orchestratormanager/Kubernetes.groovy

```diff
@@ -1819,6 +1822,10 @@ class Kubernetes implements OrchestratorMain {
     */
 
     def createDeploymentNoWait(Deployment deployment) {
+        createDeploymentNoWaitWithRetry(deployment, 0, 10)
```

Comment:
There's a `withRetry` method in the helpers `groovy/util/Helpers.groovy`. Any special reason not to use it?